rnd/autogpt_server/autogpt_server/data/graph.py

```diff
@@ -235,6 +236,36 @@ def is_input_output_block(nid: str) -> bool:
 
             # TODO: Add type compatibility check here.
 
```

Comment:
Can we do real typing rather than an any here, if I came back to this later, I’d have no idea what this returns.

Reply:
I've modified the function to now return a list of pydantic objects.